compose/foundation/foundation/src/desktopMain/kotlin/androidx/compose/foundation/text/ContextMenu.desktop.kt

```diff
@@ -130,6 +133,21 @@ private val TextFieldSelectionManager.textManager: TextManager get() = object :
     }
 }
 
+// Here we rely on the NativeClipboard directly instead of using ClipEntry,
```

Comment:
Maybe we should put this function into `ClipboardUtils` too?

Also, the same comment about using `isDataFlavorSupported` applies here too. No need to retrieve the data twice.